### PR TITLE
nm: add example of demangling C++ symbols

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: pages/osx/nm.md [hunk 1]
```diff
@@ -13,3 +13,7 @@
 - List all symbols, even debugging symbols:
 
 `nm -a {{file.o}}`
+
+- Demangle C++ symbols:
```
Comment (agnivade) [inline pages/osx/nm.md]: Can we do without the `-p` and `-i` ? If they are required / recommended, please explain what they do in the description.

Comment (ry) [reply pages/osx/nm.md]: They can be removed and I've done that in a new commit.

**Comment by sbrl** [comment]
Thanks, @ry!

**Comment by waldyrious** [comment]
See #538, where this page was introduced and the demangling behavior was discussed.

**Pull request merged**